### DISCUSSION TITLE
Codespaces cannot be reached after inactivity timeout

### DISCUSSION THREAD

**Discussion by LuisRizo** [community/community#3414] [Codespaces] [answered]
Hello there! I'm new to Discussions, so I'm not sure if this "issue" belongs to the discussions, but I have found no other place to report bugs on the Codespaces beta.

The codespace used to close automatically after inactivity and allow me to reactivate it on https://github.com/codespaces after I came back from inactivity. However, today that stopped working. I am not sure if it has anything to do with the fact that I tried using the Codespaces MacBook application, but now the Codespace is stuck in the "This site can’t be reached" state after it closes automatically due to inactivity.

All I see is "Initializing Codespace..." and then this screenshot below:

![image](https://user-images.githubusercontent.com/25105030/108773059-d145fb00-752b-11eb-9851-c573b667d7f7.png)

Same issue on the Codespaces app.

I'd be happy to provide any information that can help resolve this issue!

Thanks for this great feature!

**Comment by 2percentsilk** [comment, marked as answer]
Hi there! 
Sorry to hear you're having this issue :( 

Can you clarify what you mean by Codespaces MacBook application? We don't have a dedicated Codespaces application - rather we have an extension for VS Code that allows you to use a local installation of VS Code to connect to your codespaces. Is that what you're referring to?

**Reply by LuisRizo** [reply]
We might be referring to the same thing, but since this one showed up at the browser bar, I assumed it wasn't related to VSCode, but more to a Chrome Application (webview). I'm referring to this one:
![image](https://user-images.githubusercontent.com/25105030/108872397-220b3180-75c8-11eb-97c2-8ef1c4ef6915.png)

Also wanted to update you that seems like the issue is now resolved. I was able to re-open yesterday's codespace normally. I will update this discussion if it happens again!

Thank you for your time 🙏

**Reply by LuisRizo** [reply]
I can still reproduce the issue above every now and then after a timeout. If you'd like me to provide more info on this issue let me know, otherwise the temporary workaround is to delete and re-create the CodeSpace.

**Reply by 2percentsilk** [reply]
Thanks for the details!
That installation button is turning the tab into a Progressive Web App (PWA), which may have an issue with trying to resume suspended codespaces.

Can you try connecting from the (github.com/codespaces) page versus reopening the PWA and see if you get the same issue? Or rather, if you run into the issue resuming when using the PWA, try connecting via (github.com/codespaces) to see if works? That will help me narrow down whether it's a general resume issue or specifically an issue with the PWA implementation.
